lib/ui/alert_detail.dart

```diff
@@ -127,11 +127,19 @@ class _AlertDetailPageState extends State<AlertDetailPage> {
                         height: 20,
                         color: Theme.of(context).colorScheme.background,
                       ),
-                      FormDatePicker(
-                        date: date,
-                        onChanged: (value) {
+                      FormDateAndTimePicker(
+                        date: nextNotifyDate,
+                        time: TimeOfDay(
+                            hour: nextNotifyTime.hour,
+                            minute: nextNotifyTime.minute),
+                        dateOnChanged: (value) {
+                          setState(() {
+                            nextNotifyDate = value;
+                          });
+                        },
+                        timeOnChanged: (value) {
                           setState(() {
-                            date = value;
+                            nextNotifyTime = value;
```

Comment:
Oh I see, if we have only have one DateTime variable, we probably need to do something like:
`nextNotifyDate = nextNotifyDate.copyWith(hour: nextNotifyTime.hour, ...)`
It's probably because we have separate pickers for date and time from FormDateAndTimePicker. If we could have design like after date picker is dismiss, we popup the time picker right away, then maybe it's easier to have one single variable to track.

Reply:
i agree, that would be more ideal, I created a ticket for this improvement, we can decide if we want this change for MVP or not later. https://www.notion.so/76f32de0991c4f0ca592d3ba9e175417?v=8f65456e0b9c4a2dacafbf964540a7f4&p=fdb00a797c99485eb32c7b083e843975&pm=s